aws/src/integration/java/org/apache/iceberg/aws/glue/GlueTestBase.java

```diff
@@ -69,8 +69,8 @@
 
   // table location properties
   static final Map<String, String> tableLocationProperties = ImmutableMap.of(
-      TableProperties.WRITE_DATA_LOCATION, "s3://" + testBucketName + "/writeDataLoc",
       TableProperties.WRITE_METADATA_LOCATION, "s3://" + testBucketName + "/writeMetaDataLoc",
+      TableProperties.WRITE_DATA_LOCATION, "s3://" + testBucketName + "/writeDataLoc",
```

Comment:
if we were to make changes I would recommend changing this : 
https://github.com/apache/iceberg/blob/5b9cd7a68478f9fddd569765bfaec285d92f5907/aws/src/integration/java/org/apache/iceberg/aws/glue/TestGlueCatalogTable.java#L77

sort both the collections as it may happen post picking your changes it failing in my machine
```java
    Assert.assertEquals("additionalLocations should match", tableLocationProperties.values().stream().sorted().collect(
        Collectors.toList()),
        response.table().storageDescriptor().additionalLocations().stream().sorted().collect(Collectors.toList()));
```

Reply:
Thanks for the suggestion. That makes sense as we are comparing a map's valueSet and a list. I have updated the change. Please have another look.